CONTRIBUTING.md

```diff
@@ -95,36 +147,64 @@ The unit test suite can be run locally with:
 pytest
 ```
 
-### Documentation
+## Pull Requests (PRs)
 
-The documentation is built on CI, and is hosted by readthedocs. If you have
-modified the docstrings or notebooks, please also check that the changes are
-rendered properly in the generated HTML files.
+### Etiquette for creating PRs
 
-To build the documentation locally:
+Before starting on a PR, please make a proposal by **opening an Issue**,
+checking for any duplicates. This isn't necessary for trivial PRs such as fixing
+a typo.
 
-1. Navigate to the `docs` directory
-2. Run `make html`
-3. Open "_build/html/index.html" in your browser
+**Keep the scope small**. This makes PRs a lot easier to review. Separate
+functional code changes (such as bug fixes) from refactoring changes (such as
+style improvements). PRs should contain one or the other, but not both.
 
-## Pull Requests (PRs)
+Open a **Draft PR** as early as possible, do not wait until the feature is
+ready. Work on a feature branch with a descriptive name such as
+`fix/lightgbm-warnings` or `doc/contributing`.
 
-### Etiquette for creating PRs
+Use a descriptive title, such as:
 
-- Before starting on a PR, please make a proposal by opening an Issue, and check
-  for any duplicates. This isn't necessary for trivial PRs such as fixing a
-  typo.
-- Work on a feature branch with a descriptive name such as
-  `fix/lightgbm-warnings` or `doc/contributing`.
-- Open a Draft PR as early as possible, do not wait until the feature is ready.
-- Separate functional code changes (such as bug fixes) from refactoring changes.
-  PRs should contain one or the other, but not both.
+- `FIX: Update parameters to remove DeprecationWarning in TreeExplainer`
+- `ENH: Add support for python 3.11`
+- `DOCS: Fix formatting of ExactExplainer docstring`
 
 ### Checklist for publishing PRs
 
 Before marking your PR as "ready for review" (by removing the `Draft` status),
 please ensure:
 
 - Your feature branch is up-to-date with the master branch,
-- All [pre-commit hooks](https://pre-commit.com/#install) pass, and
+- All [pre-commit hooks](#code-checks-with-precommit) pass, and
 - Unit tests have been added (if your PR adds any new features or fixes a bug).
+
+## Documentation
+
+The documentation is hosted by at
+[shap.readthedocs.io](https://shap.readthedocs.io/en/latest/index.html). If you
+have modified the docstrings or notebooks, please also check that the changes
+are rendered properly in the generated HTML files.
+
+### Previewing changes on Pull Requests
+
+The documentation is built automatically on each Pull Request, to facilitate
+previewing how any changes will render. To see the preview:
```

Comment:
how about

"previewing how your changes will render"

"any changes" reads a little funny to me